packages/date-picker/src/DatePicker/DatePicker.spec.tsx

```diff
@@ -1082,6 +1082,46 @@ describe('packages/date-picker', () => {
     });
 
     describe('Keyboard navigation', () => {
+      describe('chaning the month back to current value month focuses the current value', () => {
+        test('after pressing the right chevron', async () => {
+          const { openMenu } = renderDatePicker({
+            value: testToday,
+          });
+
+          const { rightChevron, queryCellByDate } = await openMenu();
+
+          expect(queryCellByDate(testToday)).toHaveFocus();
+
+          rightChevron?.focus();
+          userEvent.keyboard(`[Enter]`);
+          // cell, leftChevron
+          tabNTimes(2);
+          userEvent.keyboard(`[Enter]`);
+          // select, select, rightChevron, cell
+          tabNTimes(4);
+          expect(queryCellByDate(testToday)).toHaveFocus();
+        });
```

Comment:
Is there a concern that this won't work in one of these cases (L/R chevron)? Otherwise I'd say we test "when month returns to value's month", regardless of _how_ we return to that month

IMO this should be 
- (set value)
- Open menu
- Change month with select
- Change month back
- expect highlight to be value

Reply:
I was being extra cautious but I can reduce this to one test